packages/elements/src/photon-multirx-form/components/AddPrescriptionCard.tsx

```diff
@@ -234,6 +241,8 @@ export const AddPrescriptionCard = (props: {
                   value: e.detail.catalogId
                 });
               }
+
+              props.draftedPrescriptionChanged();
```

Comment:
and also screen here when we change the medicine type of a drafted prescription (that way we can surface contraindications sooner)